tools/crates-validator/Cargo.toml

```
@@ -9,11 +9,14 @@ anyhow = "1.0"
 clap = { version = "4.0", features = ["derive"] }
 colored = "2"
 db-dump = "0.4"
+tokio-stream = "0.1.14"
 handlebars = "4.3"
-reqwest = { version = "0.11", features = ["blocking"] }
+indicatif = "0.15.0"
+reqwest = { version = "0.11.3", features = ["stream"] }
```

When possible, it's better to keep the dependencies to their minor version.

```suggestion
indicatif = "0.15"
reqwest = { version = "0.11", features = ["stream"] }
```

---

Oh, thank you!